cmip_model_rmse/OutputArtifacts.toml

```diff
@@ -0,0 +1,6 @@
+[cmip_model_rmse]
+git-tree-sha1 = "1f6d1c2e5a5fc26c4efd713245ab60ec067bfa73"
```

Comment:
What is the total size?

Maybe we want these to be lazy

Reply:
It's 4 KB in total

Reply:
Okay, it doesn't need to be lazy